Tests/Tests.iOS/Tests.iOS.csproj

```diff
@@ -127,6 +127,7 @@
     <Reference Include="System" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
+    <Reference Include="System.Runtime.Serialization" />
```

Comment:
I'm a bit worried about this - if our generated code doesn't compile on a default project, it's going to be annoying.

Reply:
What was the resolution here - did it turn out we don't need it or?

Reply:
It seems this is not an issue anymore. I suppose that I was doing something wrong with the packaging before, as the project is building without issues now